spec/pathwatcher-spec.coffee

```diff
@@ -114,6 +114,21 @@ describe 'PathWatcher', ->
           done()
       fs.writeFileSync(tempFile, 'changed')
 
+  describe 'when watching a file that does not exist', ->
+    it 'throws an error with a code', (done) ->
```

Comment:
I don't think you need to worry about the `done` callback here, this test is synchronous, or am I missing something?